### PR TITLE
Added a link saying "Help us translate this app" in the about page.

### PR DESCRIPTION
**Pull request by archisman-panigrahi**
Added a link saying "Help us translate this app" in the about page. The link points to weblate.

### CODE REVIEW

File: safeeyes/config/locale/de/LC_MESSAGES/safeeyes.po [hunk 1]
```diff
@@ -549,7 +553,8 @@ msgstr ""
 # plugin/limitconsecutiveskipping
 #, python-format
 msgid "Skipped or postponed %(num)d/%(allowed)d breaks in a row"
-msgstr "%(num)d/%(allowed)d Pausen hintereinander übersprungen oder aufgeschoben"
+msgstr ""
+"%(num)d/%(allowed)d Pausen hintereinander übersprungen oder aufgeschoben"
```
Comment (archisman-panigrahi) [inline safeeyes/config/locale/de/LC_MESSAGES/safeeyes.po:557]: @deltragon I ran update-po.sh, and this automatically happened. I am unsure why

Comment (deltragon) [reply safeeyes/config/locale/de/LC_MESSAGES/safeeyes.po:557]: Yeah, this is just formatting.

**Review by deltragon — APPROVED** [review]
LGTM.
I'd personally prefer the "Close" button to stay in the corner, and not in the center, but I'm fine either way.

**Comment by archisman-panigrahi** [comment]
The two links did not look very next to each other, that is why I brought the close button in between.

**Pull request merged**